dist/src/main/java/io/camunda/application/StandaloneCamunda.java

```diff
@@ -28,8 +25,7 @@
 public class StandaloneCamunda {
 
   private static final String SPRING_PROFILES_ACTIVE_PROPERTY = ACTIVE_PROFILES_PROPERTY_NAME;
-  private static final String DEFAULT_CAMUNDA_PROFILES =
```

Comment:
@mo-okocha let's not change StandaloneCamunda, I believe this changes were done for development tests, let's revert it.